google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -535,7 +538,8 @@ TEST_F(DataTypeIntegrationTest, SelectIntervalFromTimestampDiff) {
           std::chrono::hours(1))};
   std::time_t now_seconds =
       std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
```

Comment:
I know you didn't originate this code, but adding a units designator to a variable name is a sure-fire indication that things have already gone awry.

And time instants should not even have units in the first place (apart from, perhaps, to indicated resolution, which isn't important here).